api/bin/chainflip-broker-api/src/main.rs

```diff
@@ -309,6 +315,14 @@ impl RpcServer for RpcServerImpl {
 			.cf_get_affiliates(self.api.state_chain_client.account_id(), None)
 			.await?)
 	}
+
+	async fn affiliate_withdrawal_request(
+		&self,
+		affiliate_short_id: AffiliateShortId,
```

Comment:
In the other methods with affiliates (like `request_swap_parameter_encoding`) we take the affiliate's AccountId. I think it would be better to stay consistent. 


(Even better, we could accept one or the other, ie. `enum Affiliate { ShortId(..), LongId(..) }`. But better to leave this for a future PR I think.)